fp.js

```diff
@@ -1,28 +1,74 @@
 // Convert the following code to use `.map`
 
-var peoplesAges = []
+// var peoplesAges = []
 var people = [
   {name: "Jane", age: 32},
   {name: "Jessica", age: 34},
   {name: "Jasmine", age: 39}
 ]
-people.forEach(function(person){
-  peoplesAges.push(person.age)
-});
+// people.forEach(function(person){
+//   peoplesAges.push(person.age)
+// });
+//THis is the template from my notes on the subject.
+// var numbers = [1, 4, 9, 3, 27, 7];
+// var roots = numbers.map(Math.sqrt);
+//this is what I found on w3schools for an almost identical example, except they do not use the .map:
+// var persons = [
+//     {firstname : "Malcom", lastname: "Reynolds"},
+//     {firstname : "Kaylee", lastname: "Frye"},
+//     {firstname : "Jayne", lastname: "Cobb"}
+// ];
+//
+//
+// function getFullName(item, index) {
+//     var fullname = [item.firstname,item.lastname].join(" ");
+//     return fullname;
+// }
+//
+// function myFunction() {
+//     document.getElementById("demo").innerHTML = persons.map(getFullName);
+// }
+var peoplesAges = people.map(people.age);
+//So I don't know why my array isn't populated.
+
+
 
 // Convert the following code to use `.filter`
 
-var peopleYoungerThan35 = []
+// var peopleYoungerThan35 = []
 var people = [
   {name: "Jane", age: 32},
   {name: "Jessica", age: 34},
   {name: "Jasmine", age: 39}
 ]
-people.forEach(function(person) {
-  if(person.age < 35){
-    peopleYoungerThan35.push(person)
-  }
-});
+// people.forEach(function(person) {
+//   if(person.age < 35){
+//     peopleYoungerThan35.push(person)
+//   }
+// });
+// Here is the documentation from MDN:
+// function isBigEnough(value) {
+//   return value >= 10;
+// }
+//
+// var filtered = [12, 5, 8, 130, 44].filter(isBigEnough);
+// // filtered is [12, 130, 44]
+// I modelled the function on that:
+function youngerThan35(age){
+  return age < 35
+}
+var peopleYoungerThan35 = people.filter(youngerThan35)
```

Comment:
Same issue as above -- `.filter` needs a function as an argument. Inside that function, we want to return whatever condition each item in `peopleYoungerThan35` needs to match. So the solution looks something like this...

```js
var peopleYoungerThan35 = people.filter(function(person){
  return person.age < 35
})
```